submit/static/css/submit.css

```diff
@@ -7,4 +7,8 @@
 .is-short-field {
   max-width: 25rem; }
 
+.form-margin {
```

Comment:
```
.form-margin
  margin: .4em 0
```

belongs in submit.sass and compiled into css; indentation matters (two spaces). Ems for scaling if user increases browser text size.